elements/directory/l10nvocabulary/nuxeo-l10nvocabulary-edit-layout.html

```diff
@@ -0,0 +1,106 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!--
+`nuxeo-l10nvocabulary-edit-layout`
+@group Nuxeo UI
+@element nuxeo-l10nvocabulary-edit-layout
+-->
+<dom-module id="nuxeo-l10nvocabulary-edit-layout">
+  <template>
+    <style>
```

Comment:
Missing `nuxeo-styles` include?
```suggestion
    <style include="nuxeo-styles">
```